app/src/main/java/io/pslab/fragment/FAQFragment.java

```diff
@@ -25,128 +31,34 @@ public static FAQFragment newInstance() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        questions = getResources().getStringArray(R.array.faq_questions);
-
-        String[] ans = getResources().getStringArray(R.array.faq_answers);
-        answers = new String[ans.length][];
-        for (int i = 0; i < ans.length; i++) {
-            answers[i] = new String[]{ans[i]};
-        }
-    }
-
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        View rootView;
-        rootView = inflater.inflate(R.layout.fragment_faq, container, false);
-        return rootView;
     }
 
-    @Override
-    public void onViewCreated(View view, Bundle savedInstanceState) {
-        super.onViewCreated(view, savedInstanceState);
-        ExpandableListView listView;
-
-        listView = (ExpandableListView) view.findViewById(R.id.expListView);
-        listView.setAdapter(new ExpandableListAdapter(questions, answers));
-        listView.setGroupIndicator(null);
-
-    }
-
-    public class ExpandableListAdapter extends BaseExpandableListAdapter {
```

Comment:
Since we are not using such adapter class anywhere else in the app, should we really take it out from here and create as a separate class?

Reply:
> Since we are not using such adapter class anywhere else in the app, should we really take it out from here and create as a separate class?

Sir if you want i can put this adapter in FAQFragment.java as a nested class rather than crreate a seperate adapter but i think  this type of implementation hard to maintain

Reply:
I wonder why it's hard to maintain if the only place the class is used is inside FAQFragment. What else changed in this PR except moving the inner class to a separate outer class?

Reply:
actually previous adapter has some problem when we click on expandable listview the app was crash. #2126  please sir check it here i provide ss and gif.Sir i also refactored code and put this adapter inside FAQFragment ok.